R/model.R

```diff
@@ -331,9 +336,13 @@ compile_method <- function(quiet = TRUE,
     exe_suffix <- paste0("_", exe_suffix)
   }
   if (all(nzchar(self$exe_file()))) {
-    exe <- cmdstan_ext(paste0(strip_ext(self$stan_file()), exe_suffix))
+    if (is.null(private$dir_)) {
+      stan_file <- self$stan_file()
+    } else {
+      stan_file <- file.path(private$dir_, basename(self$stan_file()))
```

Comment:
Do we need to wrap `private$dir_` in `absolute_path()`? We might not (maybe that will already happen later to `stan_file`?)

Reply:
Good call yes! Absolute path should be used.